internal/model/plugin.go

```diff
@@ -18,9 +19,29 @@ type Plugin struct {
 	Signature       string                    `json:"signature"` // A signature of a plugin saved in base64 encoding.
 	Manifest        *mattermostModel.Manifest `json:"manifest"`
 	Enterprise      bool                      `json:"enterprise"` // Indicated if the plugin is an enterprise plugin
+	Platforms       PlatformBundles           `json:"platforms"`
 	UpdatedAt       time.Time                 `json:"updated_at"` // The point in time this release of the plugin was added to the Plugin Marketplace
 }
 
+// PlatformBundleMetadata holds the necessary data to fetch and verify a plugin built for a specific platform
+type PlatformBundleMetadata struct {
+	DownloadURL string `json:"download_url"`
+	Signature   string `json:"signature"`
+}
+
+type PlatformBundles struct {
+	LinuxAmd64   *PlatformBundleMetadata `json:"linux-amd64,omitempty" yaml:"linux-amd64,omitempty"`
+	DarwinAmd64  *PlatformBundleMetadata `json:"darwin-amd64,omitempty" yaml:"darwin-amd64,omitempty"`
+	WindowsAmd64 *PlatformBundleMetadata `json:"windows-amd64,omitempty" yaml:"windows-amd64,omitempty"`
```

Comment:
Does this need to be a pointer? Can we simplify by using a struct directly?

Reply:
It was to make it simple to verify presence of the bundle. Are you implying we do this:
```suggestion
	LinuxAmd64   PlatformBundleMetadata `json:"linux-amd64,omitempty" yaml:"linux-amd64,omitempty"`
	DarwinAmd64  PlatformBundleMetadata `json:"darwin-amd64,omitempty" yaml:"darwin-amd64,omitempty"`
	WindowsAmd64 PlatformBundleMetadata `json:"windows-amd64,omitempty" yaml:"windows-amd64,omitempty"`
```

Then when checking for a given platform's presence, we do something like `if platforms.LinuxAmd64.URL != ""` rather than `if platforms.LinuxAmd64 != nil`?

Reply:
Yes, that was my suggestion